From 2359230c8d2182ecfdf9815521d78d616a902ae0 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Mon, 1 Feb 2016 21:34:43 +0800
Subject: [PATCH 0753/1240] Revert "neta: a3700: enable "always init ETH PHY"
 feature in defconfig"

This reverts commit 5e9ce5f16e2c98b3baa8707960a97db451be5dbd.
It is because the Linux now supports Marvell 1512 serial PHY,
thus no need to always initialize the PHY in u-boot.

Change-Id: I90207b727a7e1d5b34f8e0d39fb2be8102e63605
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27097
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 configs/mvebu_armadalp_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/mvebu_armadalp_defconfig b/configs/mvebu_armadalp_defconfig
index 43d59b3..14c3de7 100644
--- a/configs/mvebu_armadalp_defconfig
+++ b/configs/mvebu_armadalp_defconfig
@@ -30,7 +30,6 @@ CONFIG_SCSI_AHCI_PLAT=y
 +S:CONFIG_MVEBU_MBUS_SPL_ONLY=y
 CONFIG_MVNETA=y
 CONFIG_MVEBU_NETA_BYPASS_DEC_WIN=y
-CONFIG_MVEBU_NETA_ALWAYS_INIT_PHY=y
 CONFIG_PHY_MARVELL=y
 CONFIG_I2C_MV=y
 # CONFIG_I2C_MV_PAD_REG is not set
-- 
1.9.1

